substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSectionImpl.java

```diff
@@ -257,6 +258,7 @@ protected int putSLEB(long val, byte[] buffer, int p) {
         return pos;
     }
 
+    @SuppressWarnings("unused")
```

Comment:
So does that mean `putAsciiStringBytes` is no longer necessary at all? In that case please remove it.

Otherwise, `putAsciiStringBytes` needs to have a verification that the string really only contains Ascii characters, to prevent any wrong usage in the future.

Reply:
Yes, having exercised the UTF8 code I now think there will be no need for `putAsciiStringBytes` so I have deleted it.